lib/compat/wordpress-6.0/class-wp-theme-json-gutenberg.php

```diff
@@ -42,4 +42,46 @@ public function get_patterns() {
 		}
 		return array();
 	}
+
+	/**
+	 * Returns a valid theme.json for a theme.
+	 * Essentially, it flattens the preset data.
+	 *
+	 * @return array
+	 */
+	public function export_theme_json() {
```

Comment:
Not sure if I like the "export" in the naming. Alternative naming:

- `get_data` as opposed to `get_raw_data` (which doesn't flatten the presets)
- `get_data_as_theme_json` to communicate that this returns the data in a way that is like a `theme.json` structure?

Reply:
The `export_theme_json` looks like an action name and not a "getter" 😅  I think we should also avoid `theme_json` in method names since that's implied from the class name.

The `get_data` or `get_flattened_data` works for me.

Reply:
Renamed to `get_data` at https://github.com/WordPress/gutenberg/pull/39048/commits/5a64ea43b290b8ee41bdf0d263fe2ae439729467